xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -3142,7 +3142,7 @@ glusterd_add_volume_to_dict(glusterd_volinfo_t *volinfo, dict_t *dict,
     if (ret)
         goto out;
 
-    snprintf(key, sizeof(key), "%s.ckusm", pfx);
+    snprintf(key, sizeof(key), "%s.cksum", pfx);
```

Comment:
On second thought, even though this name is incorrect, changing it wouldn't cause backward compatibility issues ?